Microsoft.Dynamics365.UIAutomation.Api/Pages/Navigation.cs

```diff
@@ -96,8 +96,21 @@ public BrowserCommandResult<bool> OpenSubArea(string area, string subArea, int t
                     throw new InvalidOperationException($"No subarea with the name '{subArea}' exists inside of '{area}'.");
                 }
 
-                subAreas[subArea].Click();
 
+                try
+                {
+                    subAreas[subArea].Click();
```

Comment:
I haven't stepped through this logic but if you check the attributes on this IElement, does the hidden attribute return true?   So rather than handling an exception we can look for hidden state and either make it visible then click or click with javascript?   There could be other exceptions that are thrown not specific to the element being hidden (Stale, Does not exist).